try:
    print('HELLO')
    print(WORLD)
except Exception as e:
    print('ERROR', e)
